nuad/constraints.py

```diff
@@ -2283,7 +2283,7 @@ class Strand(Part, JSONSerializable):
     or 384-well plates.
     """
 
-    _name: str | None = None
+    name: str

```

Comment:
There should be an error since `name` is being declared after `vendor_fields`, since non-optional fields need to precede optional fields